src/app/api/onboarding/route.ts

```diff
@@ -1,14 +1,41 @@
 import { formSchema } from "@/shared/onboarding";
 import { NextResponse } from "next/server";
+import { prisma } from "@/shared/db";
 
 export async function POST(request: Request) {
   try {
-    const data = await request.json();
+    const dataWithId = await request.json();
+    const { data } = dataWithId;
+    const { id } = dataWithId;
```

Comment:
Why do we need this ID? Can't we get it from the logged user?

Reply:
Yes! Thats the way.